pages/products/index.tsx

```diff
@@ -1,23 +1,43 @@
-import { Button, Dropdown, Panel, Small, StatefulTable, Link as StyledLink } from '@bigcommerce/big-design';
+import { Button, Dropdown, Panel, Small, Link as StyledLink, Table, TableSortDirection } from '@bigcommerce/big-design';
 import { MoreHorizIcon } from '@bigcommerce/big-design-icons';
 import Link from 'next/link';
 import { useRouter } from 'next/router';
-import { ReactElement } from 'react';
+import { ReactElement, useState } from 'react';
 import ErrorMessage from '../../components/error';
 import Loading from '../../components/loading';
 import { useProductList } from '../../lib/hooks';
 import { TableItem } from '../../types';
 
 const Products = () => {
+    const [itemsPerPage, setItemsPerPage] = useState(10);
+    const [currentPage, setCurrentPage] = useState(1);
+    const [columnHash, setColumnHash] = useState('id');
+    const [sortDirection, setSortDirection] = useState<TableSortDirection>('ASC');
     const router = useRouter();
-    const { error, isLoading, list = [] } = useProductList();
+    const { error, isLoading, list = [], meta = {} } = useProductList({
+      page: String(currentPage),
+      limit: String(itemsPerPage),
+      sort: columnHash,
```

Comment:
let's only set sort and direction if the user specifies a sort... we should be able to do a conditional add of a property here.
e.g.
```jsx
{
    keyOne: 'some value',
    ...(conditionOrVarToCheck && { keyTwo: 'some other value' }),
}
```